helix-lock/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same for this file. Remove all the components, which are unnecessary. Otherwise, the file is too large and imports are not required.

Reply:
I removed some that won't be needed for sure.